internal-services/manager/network_policy.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: egress-external-cluster
+spec:
+  podSelector:
+    matchLabels:
+      control-plane: controller-manager
+  policyTypes:
+  - Egress
+  egress:
+    - to:
+      - namespaceSelector: {}
+        podSelector:
+          matchLabels:
+            apiserver: "true"
+            app: openshift-kube-apiserver
+      ports:
+        - port: 6443
+        - port: 80
+        - port: 443
+          protocol: TCP
+    - to:
+      - ipBlock:
+          cidr: 44.195.126.208/32  # Hypershift-Dev
```

Comment:
will this have to change when using a dev environment each time?

Reply:
- Yes, we can either make a shared release-dev cluster to avoid the change to policy, 
- or we need to  perform `nslookup <cluster API address(without https)>`
Example: `nslookup abcd.elb.us-east-1.amazonaws.com`
on the cluster which is configured in the vault, and add the `IPAddress` from `nslookup` to `cidr` ipBlock of network policy.

Reply:
I'd prefer option 2 as long as we have a comment on the network policy yaml.

I would also add a comment to the ISC config Google doc about this.

Reply:
I have added the Network Policy config section in the [Doc](https://docs.google.com/document/d/1ElMVAgHEZ0NHcy9QAJtJAd6LrSY_n8Al18oMPE3STm0/edit#) as requested.